### PR TITLE
Howard Chen C5 calculator 1.5 PR#1

### PR DESCRIPTION
**Pull request by chenh1**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+				break;
+		}
+		$(".operDisplay").val(displayContent);
+		console.log(displayContent);
+	});
+
+	$(".calcOp").click(function(){
+
+		if(firstKeyPressed) {
+			switch (true) { //switch to see if what was pressed has a certain class
```
Comment (scbowler) [inline main.js]: This is not the proper way to use a switch, I'm surprised it appears to work. I know this is from two weeks ago so I don't know if you now have a better understanding on how to use a switch. If you are still having issues with switches or you don't understand what I mean by this is incorrect, let me know and I'll send you some examples on how to use a switch properly.

File: main.js [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+			if (calculation[1] == "") {
+				calcIndex++;
+			}
+			console.log(calcIndex);
+		}
+	});
+
+	//Squares current display
+	$(".exponent").click(function(){
+		$(".operDisplay").val(Math.pow($(".operDisplay").val(), 2));
```
Comment (scbowler) [inline main.js]: The exponent should of been handled in the switch statement that handles all of your mathematical operations. (although that switch isn't used properly either lol)

File: main.js [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+	});
+
+	//Function to change from positive to negative & vice versa.
+	$(".signInvert").click(function(){
```
Comment (scbowler) [inline main.js]: Good job switching the sign

**Comment by scbowler** [comment]
Meets all requirements for version 1.5. Well done other than the interesting use of the switch statements. :+1:

**Comment by scbowler** [comment]
lfz 2

File: main.js (hunk 1 above)
Comment (chenh1) [inline main.js]: I originally found this method of using switch from stack overflow. Dan walked me through the issue of using "true" as an expression in the switch statement. Thanks, man!

**Pull request merged**